sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -845,6 +845,30 @@ def filter_words(x):
   p.run().wait_until_finish()
 
 
+def model_textio_compressed(renames):
+  """Using a Read and Write transform to read/write text files."""
+  def filter_words(x):
+    import re
+    return re.findall(r'[A-Za-z\']+', x)
+
+  import apache_beam as beam
+  from apache_beam.utils.pipeline_options import PipelineOptions
+
+  # [START model_textio_write_compressed]
+  p = beam.Pipeline(options=PipelineOptions())
+  lines = p | 'ReadFromText' >> beam.io.ReadFromText(
+      'gs://my_bucket/path/to/input-*.csv.gz',
+      compression_type=beam.io.fileio.CompressionTypes.GZIP)
+  # [END model_textio_write_compressed]
+
+  filtered_words = lines | 'FilterWords' >> beam.FlatMap(filter_words)
```

Comment:
This, too, should not be in the snippet. Also, instead of writing, just use assert_that to verify the contents of read is correct.